cypher/refcard-tests/src/test/scala/org/neo4j/cypher/docgen/refcard/DatabaseManagementTest.scala

```diff
@@ -155,5 +163,13 @@ DROP DATABASE myDatabase IF EXISTS
 
 (★) Delete the database `myDatabase`, if it exists.
 
+###assertion=update-one
+//
+
+DROP COMPOSITE DATABASE myCompositeDatabase
+###
+
+(★) Delete the composite database `myCompositeDatabase`, throw error if it doesn't exist or isn't composite.
```

Comment:
```suggestion
(★) Delete the composite database `myCompositeDatabase`. In case this database doesn't exist or isn't composite, you will get an error.
```

Reply:
should we use `you`?

Reply:
It is not forbidden, just not advised. We can go for something like

```suggestion
(★) Delete the composite database `myCompositeDatabase`. In case this database doesn't exist or isn't composite, an error will be triggered.
```

Reply:
```suggestion
(★) Delete the composite database `myCompositeDatabase`. In case this database doesn't exist or isn't composite, an error will be thrown.
```